src/runner.ts

```diff
@@ -22,7 +23,8 @@ export interface RunDetails {
 }
 
 export class CoverageParserRunner {
-    workingDir = process.env.GITHUB_WORKSPACE + '';
+    WORKING_DIRECTORY = process.env.GITHUB_WORKSPACE + '';
+    MERGED_COBERTURA_REPORT_PATH: string = pt.join(this.WORKING_DIRECTORY, 'parasoft-merged-cobertura.xml');
 
     async run(runOptions: RunOptions) : Promise<RunDetails> {
         const parasoftReportPaths = this.findParasoftCoverageReports(runOptions.report);
```

Comment:
Currently when multiple XML files are provided but none are valid coverage reports, no error message is displayed to the user. I suggest to move the file format validation (lines 125-134) into the `findParasoftCoverageReports` method since it aligns with that method's responsibility. And the only job for `convertReportsWithJava` is to convert valid coverage reports.